docs/csharp/language-reference/keywords/ref.md

```diff
@@ -15,7 +15,8 @@ The `ref` keyword indicates a value that is passed by reference. It is used in f
 - In a method signature and in a method call, to pass an argument to a method by reference. See [Passing an argument by reference](#passing-an-argument-by-reference) for more information.
 - In a method signature, to return a value to the caller by reference. See [Reference return values](#reference-return-values) for more information.
 - In a member body, to indicate that a reference return value is stored locally as a reference that the caller intends to modify or, in general, a local variable accesses another value by reference. See [Ref locals](#ref-locals) for more information.
-- In a `struct` declaration to declare a `ref struct` or a `ref readonly struct`. For more information, see [Reference semantics with value types](../../reference-semantics-with-value-types.md).
+- In a `struct` declaration to declare a `ref struct` or a `ref readonly struct`. See [ref struct types](#ref-struct-types) for more information.
```

Comment:
The boilerplate sentence is like the original (I think even Acrolinx will tell you about that)
For more information, see x.